City-of-Helsinki/drupal-module-helfi-api-base
# UHF-9466, UHF-10012: Simplify api and vault accounts, user expire feature, remove debug api feature

## How to install

- `composer require drupal/helfi_api_base:dev-UHF-10012`
- `drush updb`
- `drush cex`

## How to test
- Make sure the debug rest endpoint and role is removed after you export the configuration
- Go to `/admin/people` and check if you have accounts that have not logged in over 6 months, if not, run `drush sqlq "update users_field_data set access = 1 where uid = 5;` (replace uid 5 with a real UID)
- Run `drush cron` and make sure accounts that have no logged in over 6 months are blocked
